src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
```

Comment:
The `random` instance variable should be declared as `final` since it is not intended to be reassigned. This ensures the instance is thread-safe and can be safely used across multiple method calls.